pandas/io/tests/sas/test_sas.py

```diff
@@ -0,0 +1,9 @@
+import pandas.util.testing as tm
+from pandas.compat import StringIO
+from pandas import read_sas
+
+class TestSasBuff(tm.TestCase):
```

Comment:
You can call this just `TestSas` (and there needs to be a blank line after this one)